File: lib/go/internal/binary/CannotPack.go
//|
//|  Copyright The Telepact Authors
//|  SPDX-License-Identifier: Apache-2.0
//|

package binary

// CannotPack indicates that a value cannot be represented in the binary encoding.
type CannotPack struct{}

// Error implements the error interface.
func (CannotPack) Error() string {
	return "cannot pack value"
}

File: lib/ts/src/internal/binary/CannotPack.ts
//|
//|  Copyright The Telepact Authors
//|  SPDX-License-Identifier: Apache-2.0
//|

export class CannotPack extends Error {
    constructor(message?: string) {
        super(message);
        Object.setPrototypeOf(this, new.target.prototype);
    }
}
